earthfile2llb/listener.go

```diff
@@ -395,7 +397,7 @@ func (l *listener) ExitSaveArtifact(c *parser.SaveArtifactContext) {
 		l.err = fmt.Errorf("no arguments provided to the SAVE ARTIFACT command")
 		return
 	}
-	if fs.NArg() > 5 {
+	if fs.NArg() > 6 {
```

Comment:
This change seems incorrect? The extra flag does not contribute to `NArg()`

Reply:
You are correct. I think this is an artifact from before I pushed EOF into the NL...

Reply:
With `?` :
![With question mark](https://user-images.githubusercontent.com/1056816/103569194-1c288800-4e84-11eb-89be-a157b64ea53c.png)

Without `?`:
![without question mark](https://user-images.githubusercontent.com/1056816/103569209-277bb380-4e84-11eb-929f-db465e31ba19.png)

Reply:
Wrong PR? :-)

Reply:
Oh man.  Yep. Cleaned.

Anyways, yes. I didnt read docs for Narg and assumed it was total args; and it added up to 5 with keepOwn and keepTs... thats what I get for assuming